roles/openshift_node/handlers/main.yml

```diff
@@ -1,3 +1,3 @@
 ---
-- name: restart openshift-node
-  service: name=openshift-node state=restarted
+- name: restart node
+  service: name={{ product_type }}-node state=restarted
```

Comment:
@detiber OK. What I'm trying to work out is how the combos end up deciding the service scripts. My initial thought based on the change would be:

- origin + openshift = atomic-openshift-*
- origin + atomic-enterprise = atomic-openshift-*
- online + openshift = origin-*
- online + atomic-enterprise = origin-*
- enterprise + openshift = atomic-openshift-*
- enterprise + atomic-enterprise = atomic-openshift-*

But I have feeling it's wrong :smile: